packages/lime-app/Makefile

```diff
@@ -48,3 +48,53 @@ define Package/$(PKG_NAME)/postinst
 endef
 
 $(eval $(call BuildPackage,$(PKG_NAME)))
+#
+# Copyright (C) Libremesh 2017
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lime-app
+PKG_VERSION:= 3ea6d26c7c789b0224d0cf9a9f500b7a93b34c9985277d61f798e35022412741
```

Comment:
this seems swapped with PKG_HASH

Reply:
Good catch. That's why we do peer review.